lib/WebhookSignature.php

```diff
@@ -17,7 +17,8 @@
      * @param string $secret secret used to generate the signature.
      * @param int $tolerance maximum difference allowed between the header's
      *  timestamp and the current time
-     * @throws SignatureVerification if the verification fails.
+     * @throws \Stripe\Error\SignatureVerification if the verification fails.
+	 * @return bool
```

Comment:
Mind correcting the tabs versus spaces problem here?

Reply:
I just tried. I don't know why that happened. it looked right in my IDE.

Reply:
It's usually a tabwidth problem (i.e., how many spaces your tabs are set to display as like 4 versus 8). Your width is shorter in your IDE so it looks okay, but the difference is visible once you get to GitHub where tabs are rendered differently.